src/main/kotlin/community/flock/eco/workday/services/email/LeaveDayMailService.kt

```diff
@@ -2,40 +2,46 @@ package community.flock.eco.workday.services.email
 
 import community.flock.eco.workday.config.properties.MailjetTemplateProperties
 import community.flock.eco.workday.model.LeaveDay
+import community.flock.eco.workday.utils.DateUtils.toHumanReadable
 import org.slf4j.Logger
 import org.slf4j.LoggerFactory
 import org.springframework.stereotype.Service
 import java.time.format.DateTimeFormatter
 
 @Service
-class LeaveDayEmailService(private val emailService: EmailService, private val mailjetTemplateProperties: MailjetTemplateProperties) {
-    private val log: Logger = LoggerFactory.getLogger(LeaveDayEmailService::class.java)
+class LeaveDayMailService(
+    private val emailService: EmailService,
+    private val mailjetTemplateProperties: MailjetTemplateProperties,
+) {
+    private val log: Logger = LoggerFactory.getLogger(LeaveDayMailService::class.java)
 
-    fun sendUpdate(
-        old: LeaveDay,
-        new: LeaveDay,
-    ) {
-        val recipient = new.person
+    fun sendUpdate(leaveDay: LeaveDay) {
+        val recipient = leaveDay.person
 
-        var subject = "Update in Leave Day."
-        var emailMessage = "Er is een update in Leave Day."
+        val subject = "Leave Day update - ${leaveDay.description}"
```

Comment:
Als je overal Nederlandse namen gaat gebruiken, wil je dat hier dan niet ook?

Reply:
Ik denk ook dat de datum bij vrije dagen, maar ook bij ziektedagen zoals hieronder belangrijker is dan de omschrijving

Reply:
Interessant 🤔  Mijn beschrijving van leave en sickdays zijn vaak uniek genoeg. Maar we passen het aan!